### PR TITLE
Stubbing madness to get jasmine test working

### PR DESCRIPTION
**Pull request by ElvisASC4**

### CODE REVIEW

File: ui/jasmine_test/spec/helpers/stubs/StubAnalyser.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export class StubAnalyser {
+ 
+    constructor() {
+        this.frequencyBinCount = 32;
+        this.getByteFrequencyData = jasmine.createSpy("getByteFrequencyData");
```
Comment (mmallett) [inline ui/jasmine_test/spec/helpers/stubs/StubAnalyser.js:5]: nit: prefer `''` throughout for strings

File: ui/jasmine_test/spec/helpers/stubs/StubContext.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import {StubSource} from './StubSource.js';
+import {StubAnalyser} from './StubAnalyser.js';
+
+export class StubContext {
```
Comment (mmallett) [inline ui/jasmine_test/spec/helpers/stubs/StubContext.js:4]: Can you name this class `StubAudioContext` to reduce confusion, as there is also a canvas context floating around in this code

File: ui/jasmine_test/spec/helpers/stubs/StubAnalyser.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export class StubAnalyser {
```
Comment (ashleygranitto) [inline ui/jasmine_test/spec/helpers/stubs/StubAnalyser.js:1]: nit: add copyright at the top of each new file

Comment (ElvisASC4) [reply ui/jasmine_test/spec/helpers/stubs/StubAnalyser.js:1]: good catch

**Pull request merged**